resources/tentacle.rb

```diff
@@ -18,7 +18,7 @@
 # limitations under the License.
 #
 
-actions :install, :configure, :remove, :uninstall
+actions :install, :configure, :register, :remove, :uninstall
```

Comment:
Awesome. Ignore my comment about the verify methods.  I forgot this was already done for other resources.  Looks good.